tenders/hvt/hvt_core.c

```diff
@@ -224,6 +224,7 @@ static void hypercall_poll(struct hvt *hvt, hvt_gpa_t gpa)
             .tv_nsec = t->timeout_nsecs % 1000000000ULL
         }
     };
+    it.it_value.tv_nsec |= 1;
```

Comment:
I recommend a comment for this line, it is non-obvious why it is here (and not in the other cases) - until you read up on `timerfd_settime`.

Reply:
It's in the commit history, but OK, done.